### PR TITLE
Sum of nth terms

### PR DESCRIPTION
**Pull request by zachrickert**
Code from codewars.

### CODE REVIEW

File: sum_series.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def series_sum(n):
```
Comment (nhuntwalker) [inline sum_series.py]: I can haz docstrings?

File: sum_series.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    if n == 0:
+        return 0
```
Comment (nhuntwalker) [inline sum_series.py]: This should return `"0.00"` not`0`

File: sum_series.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    sum = 0
```
Comment (nhuntwalker) [inline sum_series.py]: `sum` is a built-in function and should not be reassigned.

File: sum_series.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    for i in range(n):
+        sum += 1 / ((3 * i) + 1)
```
Comment (nhuntwalker) [inline sum_series.py]: Integer division will produce an integer as a result in Python 2. You need to make one of these numbers a `float`

File: test_sum_terms.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Test.assert_equals(series_sum(1), "1.00")
```
Comment (nhuntwalker) [inline test_sum_terms.py]: You're using a `Test` object even though `Test` exists nowhere in your file. Did you try running these in your own console?

File: src/test_sum_series.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
+"""Test  file for sum series."""
+import pytest
```
Comment (cewing) [inline src/test_sum_series.py:2]: you don't actually need this import unless you are going to use something defined by pytest, like `pytest.raises` or `pytest.fixture` or `pytest.mark.parametrize`.

**Pull request merged**